### ISSUE TITLE
Validator Stopped Error `Process exited with SIGINT`

### ISSUE THREAD

**Issue by khyldo** [Shardeum/shardeum-bug-reporting#306]
I have the following text from my validator when i execute the exit-summary.json command : 

```json
{
  "status": "Exit with warning",
  "message": "Process exited with SIGINT",
  "exitTime": 1680711713354,
  "totalActiveTime": 0,
  "nodeInfo": {
    "publicKey": "f9c1b07e731c96a4ddb49ae9a68fa87b8c4b87c102c3e0d67a667de81b85dec6",
    "curvePublicKey": "1f8f0a58a3a88f837502d69ad8d225a068e635f033b4c9357e44b29753550e49",
    "externalIp": "158.220.105.188",
    "externalPort": 9001,
    "internalIp": "158.220.105.188",
    "internalPort": 10001,
    "status": null
  },
  "activeNodes": 0
}
```

Do you know what it means and how to fix it ?

**Comment by MarcusWentz** [comment]
Do you have a screenshot from the Validator GUI as well?

**Comment by khyldo** [comment]
Thanks for your answer. Is this screenshot what you asked for ? If not, plz tell me how to send you the proper screenshot

![Capture d’écran 2023-04-05 à 23 27 56](https://user-images.githubusercontent.com/114832175/230215895-a60f94f9-0c78-4ade-9d6a-e7759d25bc09.png)

**Comment by MarcusWentz** [comment]
What do you see on the Validator GUI Maintenance page
https://docs.shardeum.org/node/run/validator#step-5-start-validator
?

**Comment by khyldo** [comment]
Here is the maintenance page

![Capture d’écran 2023-04-06 à 00 32 43](https://user-images.githubusercontent.com/114832175/230226342-fa22914b-6847-4c85-820e-f92fa475a0b6.png)

**Comment by MarcusWentz** [comment]
It looks like your validator is running again now.

I noticed someone else had this issue here too:

https://github.com/Shardeum/shardeum-bug-reporting/issues/234#issuecomment-1478705524

**Comment by khyldo** [comment]
I don't understand why you say that the validator is running again now, still 0 validating time this morning, still stuck in standby, and the error is still here.

I looked at the other bug report and it looks like the answer is only : wait for an update. So this is what I have to do ? We can't solve it unless by waiting for an update ? Doesn't the 1.1.5 was supposed to fix this ?

**Comment by MarcusWentz** [comment]
Your validator is running (start mode) if it is at least in standby mode. 

There is nothing you need to do. We will continue to look into this.

**Comment by noel4za** [comment]
run 2 node.
1 node has benn 3 time active.
the other node have been more than a week in standby.
this main log 
`[2023-04-03T21:10:48.240] [INFO] main - Logger initialized.
[2023-04-03T21:10:50.064] [WARN] main - Failed to get external IP from gateway: UPnP get external failed
[2023-04-03T21:10:51.227] [INFO] main - Network: External server running on port 9001...
[2023-04-03T21:10:51.240] [INFO] main - Unable to load keypair. Generating new...
[2023-04-03T21:10:51.240] [INFO] main - New keypair generated.
[2023-04-03T21:10:51.242] [INFO] main - New keypair successfully generated and saved to database.
[2023-04-05T12:50:57.692] [INFO] main - Stopping reporter...
[2023-04-05T12:50:57.692] [INFO] main - Stopping statistics reporting...
[2023-04-05T12:50:57.692] [INFO] main - Stopping POW generators...
[2023-04-05T12:50:57.692] [INFO] main - Shutting down p2p...
[2023-04-05T12:50:57.696] [INFO] main - Shutting down the application...
[2023-04-05T12:50:57.697] [INFO] main - Shutting down networking...
[2023-04-05T12:50:57.697] [INFO] main - Closing Database connections...
[2023-04-05T12:50:57.699] [INFO] main - Shutting down logs...
[2023-04-05T12:51:00.167] [INFO] main - Logger initialized.
[2023-04-05T12:51:01.990] [WARN] main - Failed to get external IP from gateway: UPnP get external failed
[2023-04-05T12:51:02.463] [INFO] main - Network: External server running on port 9001...
[2023-04-05T12:51:07.491] [INFO] main - Keypair loaded from file secrets.json
`
exit.log
`[2023-04-05T12:50:57.692] [FATAL] exit - isCleanExit: false  exitType: SIGINT  msg: Process exited with SIGINT
[2023-04-05T12:50:57.695] [FATAL] exit - System Memory Report.  Timestamp: 1680699057693
rss: 126.48 MB
heapTotal: 57.67 MB
heapUsed: 48.68 MB
external: 34.76 MB
arrayBuffers: 16.98 MB


         0 P2P Nodelist numActiveNodes
         0 StateManager AccountsCache workingAccounts
         0 StateManager AccountsCache mainMap
         0 StateManager TXQueue queueCount
         0 StateManager TXQueue pendingQueueCount
         0 StateManager TXQueue archiveQueueCount
         0 StateManager TXQueue executeQueueLength
         1 Patcher insync true
         1 Patcher history []
         1 Patcher insync true
     0.114 Process CPU cpuPercent
         0 Process CPU cpuAVGPercent
         1 Process CPU cpu: {"min":null,"max":0,"avg":0,"allVals":[],"sum":0}
 338070081 Process Details userCPUTime
  93665623 Process Details systemCPUTime
    141664 Process Details maxRSS
         0 Process Details sharedMemorySize
         0 Process Details unsharedDataSize
         0 Process Details unsharedStackSize
  17838776 Process Details minorPageFault
         0 Process Details majorPageFault
         0 Process Details swappedOut
         0 Process Details fsRead
    233200 Process Details fsWrite
         0 Process Details ipcSent
         0 Process Details ipcReceived
         0 Process Details signalsCount
   1173218 Process Details voluntaryContextSwitches
    162349 Process Details involuntaryContextSwitches
         1 NetStats stats stats

[2023-04-05T12:50:57.696] [FATAL] exit -       3981  shardeum-staking
       825 ___ calling queryCertificate
       516 ___ lastCertTimeTxTimestamp === 0 first time or expired
       481 ___ valid cert, isReadyToJoin = true
       481 ___ calling getJoinData
       481 ___ stakeCert != null
       335 ___ call to queryCertificate failed with reason: Operator certificate has expired
       146 ___ call to queryCertificate failed with reason: node account not found
       135 ___ stakeCert expired and need to be renewed
       121 ___ call to queryCertificate failed with reason: Failed to fetch operator account state
       119 ___ stakeCert is expired or expiring soon
        95 ___ failed call to injectSetCertTimeTx 1 reason: node account not found
        68 ___ failed call to injectSetCertTimeTx 1 reason: Maximum load exceeded.
        56 ___ call to queryCertificate failed with reason: Failed to fetch node account state
        39 ___ call to queryCertificate failed with reason: undefined
        34 ___ failed call to injectSetCertTimeTx 2 reason: node account not found
        14 ___ failed call to injectSetCertTimeTx 2 reason: Maximum load exceeded.
        14 ___ failed call to injectSetCertTimeTx 1 reason: Failed to inject tx:  {}
         5 ___ call to queryCertificate failed with reason: Failed to get query certificate
         5 ___ failed call to injectSetCertTimeTx 2 reason: Failed to inject tx:  {}
         4 ___ stakeCert is expiring soon
         1 ___ failed call to injectSetCertTimeTx 3 reason: Maximum load exceeded.
         1 ___ failed call to injectSetCertTimeTx 3 reason: Failed to inject tx:  {}
         1 ___ failed call to injectSetCertTimeTx 2 reason: connect ECONNREFUSED 75.119.155.155:9001
         1 ___ failed call to injectSetCertTimeTx 1 reason: connect ECONNREFUSED 185.215.165.242:9001
         1 ___ failed call to injectSetCertTimeTx 2 reason: connect ECONNREFUSED 82.208.20.14:9001
         1 ___ failed call to injectSetCertTimeTx 1 reason: Transaction Expired
         1 ___ failed call to injectSetCertTimeTx 1 reason: connect ECONNREFUSED 94.131.14.6:9003
       481  p2p
       481 ___ join-allow-bogon-submit:false
         3  profiler
         3 ___ db
         1  network
         1 ___ init

`

**Comment by khyldo** [comment]
Little update of my situation, I did the 1.1.7 update this morning, but the SIGINT error is still here in the exit logs. Moreover, my node has stayed in standby for the past week and has never been active, not even once...

**Comment by khyldo** [comment]
New update, I tried to delete and re-install the node, and it didn't changed anything, the SIGINT error was still there on the new node. But I waited a couple days after that and it was fixed without me doing anything about it. 

Now the exit logs show a clean exit. My node also managed to become active yesterday. 

However, I continue to earn some SHM when the node is in standby and I am not sure if it's normal...
